addons/no-sprite-confirm/userscript.js

```diff
@@ -0,0 +1,8 @@
+export default async ({ addon, console }) => {
+  while (true) {
+    const confirmButton = await addon.tab.waitForElement("[class^='delete-confirmation-prompt_ok-button_']", {
+      markAsSeen: true,
+    });
```

Comment:
I think a global "click" listener ends up being more performant

Reply:
So a listener on `body` and then checking the element in it?

Reply:
Sure, it sounds reasonable to make 1 document.querySelector() call for each click made by the user.